### PR TITLE
Search result at 2237

### PR DESCRIPTION
**Pull request by vikashruhilCB**
left facet - initial structure and flow for single working component. service is for temp use - have't deleted yet, may require in future use to provide connection b/w components.

### CODE REVIEW

File: projects/cut-lib/src/lib/search/main/facets/checklist-facet/checklist-facet.component.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+}
+.filter-title {
+  font-size: 0.8rem;
+}
+.list-icons {
+  font-size: 0.9rem;
+  color: #1d5c8b;
+}
+.label-list{
+  margin: 5px 5px 5px 15px;
```
Comment (karch4162) [inline projects/cut-lib/src/lib/search/main/facets/checklist-facet/checklist-facet.component.scss]: rems please, you can change after merge, note there is a rem-calc mixin available in utilities.

File: projects/cut-lib/src/lib/search/main/facets/checklist-facet/checklist-facet.component.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+@Component({
+  selector: 'cut-checklist-facet',
+  templateUrl: './checklist-facet.component.html',
+  styleUrls: ['./checklist-facet.component.scss']
+})
+
+export class ChecklistFacetComponent implements OnInit {
+
+  //For demo purpose - default value.
+  @Input() options: CheckListModel[] = [
```
Comment (karch4162) [inline projects/cut-lib/src/lib/search/main/facets/checklist-facet/checklist-facet.component.ts]: if you need default data let's pass this in from the consuming app.  In our case the examples app.

File: projects/cut-lib/src/lib/search/main/facets/city-facet/city-facet.component.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+.filter-area p {
+  color: #767676;
+  margin: 5px 0;
+}
+.input-container {
+  flex-basis: auto;
+  flex-grow: 2;
+}
+.input-container input {
+  padding: 4px 5px;
```
Comment (karch4162) [inline projects/cut-lib/src/lib/search/main/facets/city-facet/city-facet.component.scss]: again no px please, use rems

File: projects/cut-lib/src/lib/search/main/facets/city-facet/city-facet.component.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  border-radius: 3px;
+  border-style: none;
+  border: 1px solid #767676;
```
Comment (karch4162) [inline projects/cut-lib/src/lib/search/main/facets/city-facet/city-facet.component.scss]: most theme colors are defined, lets use the color variables

**Comment by karch4162** [comment]
Your test cases are still breaking, can you please fix?

**Comment by vikashruhilCB** [comment]
> Your test cases are still breaking, can you please fix?

fixed.

**Comment by vikashruhilCB** [comment]
> Please fix the build break

fixed.

**Pull request merged**